.gitignore

```diff
@@ -3,6 +3,7 @@
 .config
 .rvmrc
 .yardoc
+.ruby-version
```

Comment:
Adding `.ruby-version` to `.gitignore` so that nobody will commit it accidentally... :D I mean I'm fine when `.ruby-version` is committed in repo, when only that ruby version is supported (e.g. for some ruby app), but overwise it's as good as adding `ruby 2.4` line into `Gemfile`.